src/multi_monitor/db.py

```diff
@@ -179,15 +182,16 @@ def create_collection(
             description: Optional description
             metadata: Optional additional metadata
             tags: Optional list of tags
+            omb_control_number: Optional OMB reference number for data collection authorities
             
         Returns:
             Collection ID (UUID)
         """
         collection_id = str(uuid.uuid4())
         self.conn.execute("""
-            INSERT INTO collections (id, name, description, metadata, tags)
+            INSERT INTO collections (id, name, description, metadata, tags, omb_control_number)
             VALUES (?, ?, ?, ?, ?)
```

Comment:
I suppose this needs an extra `?` for the new field being added, like:

```suggestion
            VALUES (?, ?, ?, ?, ?, ?)
```